CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 0.8.1 (UNRELEASED)
+---------------------------
+
+- Adds support for specifying ``kubernetes_job_timeout`` for Kubernetes compute backend jobs.
+- Adds new condition to allow to process jobs - job's containers should not be running.
```

Comment:
The first news phrase is fine, but the second is probably hard to understand out-of-the-box in the current format. What condition and why they should not be running? Perhaps try to expand a bit...

Reply:
```
- Adds a new condition to allow processing jobs because of receiving multiple failed events - job containers should not be in a running state.
```

What do you think about this version?

Reply:
Perhaps something like "Adds a new condition to allow processing jobs in case of receiving multiple failed events when job containers are not in a running state"?